.github/workflows/build-hw-reusable.yml

```diff
@@ -7,13 +7,21 @@ on:
       adapter_name:
         required: true
         type: string
+      other_adapter_name:
+        required: false
+        type: string
+        default: ""
       runner_name:
         required: true
         type: string
       platform:
         required: false
         type: string
         default: ""
+      other_platform:
+        required: false
+        type: string
+        default: ""
```

Comment:
I don't see that this value was used in the added jobs, is there a need to have this?

Reply:
I've changed it so that "platform" is now directly the value for `UR_CTS_ADAPTER_PLATFORM` rather than rebuilding it.